resources/styles/components/scores/table.less

```diff
@@ -0,0 +1,472 @@
+        fill: @pie-color-hover;
+      }
+      .pie-progress {
+        .slice {
+          fill: @pie-color;
+          mask: url(#dividers);
+          &.late {
+            fill: @pie-color-late;
+          }
+        }
+        &:hover .slice {
+          fill: @pie-color-hover;
+        }
+        .dividers rect {
+          fill: @tutor-white;
+        }
+        .backdrop {
+          fill: @pie-bg-color;
+          mask: url(#dividers);
+        }
+      }
+      .trigger-wrap {
+        padding: 10px;
+      }
+    }
+    .score {
+      #fonts > .scores-cells();
+      position: relative;
+    }
+    .late-caret-trigger {
+        position: absolute;
+        cursor: pointer;
+        top: 0px;
+        right: 0px;
+        width: 25px;
+        height: @table-row-height;
+      .late-caret {
+        position: absolute;
+        cursor: pointer;
+        top: 0px;
+        right: 0px;
+        width: 0;
+        height: 0;
+        border-style: solid;
+        border-width: 0 10px 10px 0;
+        border-color: transparent @tutor-primary transparent transparent;
+        &.accepted {
+          border-color: transparent lighten(@tutor-neutral, 25%) transparent transparent;
+        }
+      }
+    }
+  }
+  .name-cell {
+    .name {
+      text-align: left;
+      padding-left: 8px;
+    }
+  }
+
+
+
+  .filter-row {
+    width: 100%;
+    height: 70px;
+    display: block;
+    div {
+      float: left;
+    }
+    div + div {
+      margin-left: 10px;
+    }
+    .filter-group {
+      box-shadow: none;
+      button {
+        &.btn-default {
+          border-radius: 0px;
+        }
+        &.btn-primary {
+          border-radius: 0px;
+          border: @tutor-tertiary-light;
+          background: @tutor-tertiary-light;
+          height: 32px;
+          &:hover {
+            background: @tutor-tertiary-light;
+          }
+        }
+      }
+    }
+    .filter-item {
+      &:first-child {
+        width: 250px;
+      }
+      float: left;
+      display: flex;
+      align-items: center;
+      .filter-label {
+        font-size: 1.4rem;
+      }
+    }
+  }
+
+}
+
+.scores-scores-tooltip-completed-info {
+  @completed-popover-row-height: 30px;
+  height: auto;
+  .popover-content {
+    height: 100%;
+    overflow: hidden;
+    padding: 0;
+  }
+  .info {
+    display:table;
+    .row {
+      display: table-row;
+      &:nth-child(1) {
+        background: darken(@tutor-info-bg, 5%);
+      }
+      &:nth-child(2) {
+        color: @tutor-neutral;
+      }
+      &:nth-child(3) {
+        background: @tutor-neutral-lighter;
+        color: @tutor-black;
+      }
+      div {
+        font-weight: bold;
+        width: 150px;
+        height: @completed-popover-row-height;
+        vertical-align: middle;
+        display: table-cell;
+        padding-left: 20px;
+      }
+    }
+  }
+
+}
+
+.late-work-info-popover {
+  height: 98px;
+  min-width: 315px;
+  border-radius: 0;
+  border: 1px solid @tutor-neutral-light;
```

Comment:
I'm confused by why the dimensions are so precise?  Is there a reason for the `98px` and `315` vs just say `100` and `300` I'd find it clearer if we could either comment them, or assign it to a master variable and then do math as needed to express why it's 98px for instance.